app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] [OPTIONAL] Please use the callback `after_save` to call the `update_likes_counter`, so the method is called automatically every time a new post has been saved to the database, then make the `update_likes_counter` method private.